### ISSUE TITLE
NullPointerException: Attempt to invoke virtual method 'org.wordpress.android.fluxc.model.EditorThemeSupport org.wordpr...

### ISSUE THREAD

**Issue by AliSoftware** [wordpress-mobile/WordPress-Android#15300]
- **Sentry Urls**: (events/users)
  - https://sentry.io/share/issue/67078fb4bbe14c8c8380f4114a2be02c/ (26/1)
  - https://sentry.io/share/issue/d38dd0d3aa4743619ea43a1655c0cd90/ (21/3)
  - https://sentry.io/share/issue/d1e705d510174c5fbba40142b05cf735/ (5/1)
  - https://sentry.io/share/issue/ca3efba9846147308537b7df17ed8d59/ (4/1)
  - https://sentry.io/share/issue/9622c255077b442d9e11c5f787c25a7d/ (2/1)
- **User Count**: 1 + 3 + 1 + 1 + 1 = 7
- **Count**: 26 + 21 + 5 + 5 + 2 = 59
- **First Release**: 18.2-rc-1
- **First Seen**: 2021-09-06T17:53:57.558000Z
- **Last Seen**: 2021-09-08T09:55:45Z
- **24 Hours**: 15 _(value extracted only from the first issue with 26/1)_
- **30 Days**: 26 _(value extracted only from the first issue with 26/1)_

```
NullPointerException: Attempt to invoke virtual method 'org.wordpress.android.fluxc.model.EditorThemeSupport org.wordpress.android.fluxc.model.EditorTheme.getThemeSupport()' on a null object reference
     at org.wordpress.android.ui.posts.EditPostActivity.onEditorFinalTouchesBeforeShowing(EditPostActivity.java:3305)
     at org.wordpress.android.ui.posts.EditPostActivity.onEditorFragmentInitialized(EditPostActivity.java:3283)
     at org.wordpress.android.editor.AztecEditorFragment.onCreateView(AztecEditorFragment.java:382)
...
(56 additional frame(s) were not displayed)
```

There seem to be multiple Sentry issues all with the similar exception for this one. Might be worth considering merging them (after double-checking they are indeed the same)

**Comment by develric** [comment]
Hey @AliSoftware 👋 

>There seem to be multiple Sentry issues all with the similar exception for this one. Might be worth considering merging them (after double-checking they are indeed the same)

I agree with you they are the same issue. 

The issue seems to be that [mEditorThemeStore.getEditorThemeForSite](https://github.com/wordpress-mobile/WordPress-Android/blob/a369e38db6279e8349554177cc494141b2248ccd/WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java#L3304) can return a null value that we do not check for.

Current beta users count are not huge but we get an impact of 1.8% (11 affected users / 589 total users) and it's a crash, so I'm assigning an High priority, and I think we should fix it in the current beta.

It seems to have been introduced [here](https://github.com/wordpress-mobile/WordPress-Android/pull/15134). Hi @antonis 👋 , do you think you could have a look here? Let me know if someone else should look into it 🙇 .

>Might be worth considering merging them

I agree; I'm going to do it as soon as I get how it's done in Sentry 😄.

**Comment by develric** [comment]
Issues were merged in sentry [into this common event](https://sentry.io/share/issue/c95cf22e82cb444eb832406ee3019bf5/).

**Comment by antonis** [comment]
Resolved in `18.2` with https://github.com/wordpress-mobile/WordPress-Android/pull/15311
